src/ol/source/wmtssource.js

```diff
@@ -428,6 +401,19 @@ ol.source.WMTS.optionsFromCapabilities = function(wmtsCap, config) {
         /urn:ogc:def:crs:(\w+):(.*:)?(\w+)$/, '$1:$3'));
   }
 
+  var projectionExtent = projection.getExtent();
+  var extent;
+  if (!goog.isNull(projectionExtent)) {
```

Comment:
I don't understand the logic here. It should surely always use the caps BBox whether there's a projection extent or not.